exercises/practice/square-root/square-root.jl

```diff
@@ -0,0 +1,3 @@
+function square_root(n::Int64; tol)
+    
+end
```

Comment:
Stubs should end with a blank line, I think, to match unix expectations.

```suggestion
end

```